packages/api/tests/api/users.ts/me/index.test.ts

```diff
@@ -0,0 +1,31 @@
+import express, { Request, Response } from 'express';
+import supertest from 'supertest';
+import { createMockNext } from '../../../testUtils/createMockNext';
+import { getMock } from '../../../testUtils/getMock';
+import { mountUserMiddleware } from '../../../../src/middleware/mountUserMiddleware';
+
+jest.mock('../../../../src/api/users/me/get', () => ({
+  get: (req: Request, res: Response) => {
+    res.sendStatus(200);
+  },
+}));
+
+jest.mock('../../../../src/middleware/mountUserMiddleware', () => ({
+  mountUserMiddleware: createMockNext(),
```

Comment:
This is mocking an unused method; you needed to mock `validateSessionMountUser` - I'm gonna clean that up to make it more obvious but for now just swap the mocked method here and your test should pass